WooCommerce/Classes/ViewRelated/Themes/ThemesPreviewView.swift

```diff
@@ -126,6 +140,56 @@ struct ThemesPreviewView: View {
             }
             .navigationBarTitleDisplayMode(.inline)
         }
+        .task { await viewModel.fetchPages() }
+        .sheet(isPresented: $showPagesMenu) {
+            pagesListSheet(pages: viewModel.pages)
+        }
+    }
+
+    private func pageSelector(state: ThemesPreviewViewModel.State, pageTitle: String) -> some View {
+        switch state {
+        case .pagesLoading:
+            return AnyView(Text(Localization.preview))
+
+        case .pagesContent:
+            return AnyView(
+                Button(action: { showPagesMenu = true },
+                       label: {
+                           HStack {
+                               Text(pageTitle).bodyStyle()
+                               Image(uiImage: .chevronDownImage)
+                                   .fixedSize()
+                                   .secondaryBodyStyle()
```

Comment:
The theme picker dropdown downfacing arrow should be black color. The current color looks gray. 
<img width="155" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/524475/7c68e51c-761e-497b-aee3-31e91ddbd18e">

Reply:
Good eye! https://github.com/woocommerce/woocommerce-ios/pull/11492/commits/754cd51f32bbadee86cb935d996a2dd98cc8702b